osu.Framework/Platform/GameHost.cs

```
@@ -798,7 +798,7 @@ protected virtual void SetupConfig(IDictionary<FrameworkSetting, object> gameDef
             bypassFrontToBackPass = DebugConfig.GetBindable<bool>(DebugSetting.BypassFrontToBackPass);
 
             threadLocale = Config.GetBindable<string>(FrameworkSetting.Locale);
-            threadLocale.BindValueChanged(locale =>
+            threadLocale?.BindValueChanged(locale =>
```

Config bindables should never be null. This should probably return an exception instead on the `GetBindable()` call.